index.html

```diff
@@ -16,13 +17,17 @@
         <nav>
          <!-- navigation bar begins here -->
             <a href="#"><h2 class="logo">Diana</h2></a> 
-            <ul>
-                <li> <a href="#">Portoflio</a></li> 
-                <li> <a href="#">About</a></li>
-                <li> <a href="#">Contact</a></li>
-                <li><a href=""><img src="image/Union.png" alt="union-icon" class="union"></a></li>
-            </ul>         
-            <a class="hambuger" href=""><img src="image/Union.svg" alt="union-icon" ></a>            
+            <ul class="nav-menu">
+                <li> <a href="#link1" class="nav-item">Portoflio</a></li> 
+                <li> <a href="#about" class="nav-item">About</a></li>
+                <li> <a href="#link2" class="nav-item">Contact</a></li>
+                <li><a href="#link2"><img src="image/Union.png" alt="union-icon" class="union"></a></li>
+            </ul>
+            <div class="hamburger">
+                <span class="bar bar1"></span> 
+                <span class="bar bar2"></span>
+                <span class="bar bar3"></span> 
+            </div>         
```

Comment:
### Your Mobile Menu
![student-mobile-menu (copy)](https://user-images.githubusercontent.com/93039359/189873251-fca22391-bf5f-4000-b7bc-efd63cc49d0c.png)

### Figma Mobile Menu
![figma-mobile-menu (copy)](https://user-images.githubusercontent.com/93039359/189873269-56c7a750-dca6-4967-825e-cbef8d73cf9b.png)

- [ ] You can see the difference between your mobile menu and the Figma mobile menu. I have added the changes in the screenshots. Please compare both screenshots and make the changes :blush: